plugins/rikki/heroeslounge/routes.php

```diff
@@ -31,9 +31,9 @@
     Route::get('matches/withApprovedCastBetween/{startdate}/{enddate}','Rikki\Heroeslounge\Http\Match@withApprovedCastBetween');
     Route::get('matches/{id}/games','Rikki\Heroeslounge\Http\Match@games');
     Route::get('matches/{id}/replays','Rikki\Heroeslounge\Http\Match@replays');
-    Route::get('matches/today','Rikki\Heroeslounge\Http\Match@getTodaysMatches');
```

Comment:
I would like for this to stay around with day of server as default, otherwise anything that calls this will suddenly cease to work

Reply:
It does, look 2 lines down, and note the handling of "today" in the method

Reply:
It does, look 2 lines down, and note the handling of "today" in the method.  Also note my very first example above